### PR TITLE
Fix conflicts with kubed

### PR DESCRIPTION
**Pull request by mvleandro**
This PR fixes possible conflict scenarios when the cluster is already running kubed.

### CODE REVIEW

File: kube_syncer/secret_watcher.go [hunk 1]
```diff
@@ -24,7 +24,7 @@ func (w SecretWatcher) Watch() <-chan abstractions.ISynchronizable {
 	secretsChan := make(chan abstractions.ISynchronizable)
 
 	go func() {
-		watcher, _ := w.kubeClient.CoreV1().Secrets(metav1.NamespaceAll).Watch(context.Background(), metav1.ListOptions{
+		watcher, _ := w.kubeClient.CoreV1().Secrets(metav1.NamespaceAll).Watch(context.TODO(), metav1.ListOptions{
```
Comment (mvleandro) [inline kube_syncer/secret_watcher.go:27]: Background returns a non-nil, empty Context. It is never canceled, has no values, and has no deadline. It is typically used by the main function, initialization, and tests, and as the top-level Context for incoming requests.

File: kube_syncer/configmap_watcher.go [hunk 2]
```diff
@@ -24,10 +24,11 @@ func (w ConfigMapWatcher) Watch() <-chan abstractions.ISynchronizable {
 	configMapsChan := make(chan abstractions.ISynchronizable)
 
 	go func() {
-		watcher, _ := w.kubeClient.CoreV1().ConfigMaps(metav1.NamespaceAll).Watch(context.Background(), metav1.ListOptions{
+		watcher, _ := w.kubeClient.CoreV1().ConfigMaps(metav1.NamespaceAll).Watch(context.TODO(), metav1.ListOptions{
```
Comment (mvleandro) [inline kube_syncer/configmap_watcher.go:27]: Background returns a non-nil, empty Context. It is never canceled, has no values, and has no deadline. It is typically used by the main function, initialization, and tests, and as the top-level Context for incoming requests.

File: kube_syncer/abstractions/kubernetes_entity.go [hunk 3]
```diff
@@ -151,7 +163,7 @@ func (e *KubernetesEntity) Delete() error {
 		error := client.Delete(context.TODO(), entity.Name, v1.DeleteOptions{})
 
 		if error == nil {
-			Logger.Infof("The secret '%s' was delete from namespace '%s' on context '%s'.", entity.Name, entity.Namespace, e.DestinationContext)
+			Logger.Infof("The secret '%s' was deleted from namespace '%s' on context '%s'.", entity.Name, entity.Namespace, e.DestinationContext)
```
Comment (mvleandro) [inline kube_syncer/abstractions/kubernetes_entity.go:166]: Fixing log message typo.

File: kube_syncer/abstractions/kubernetes_entity.go [hunk 4]
```diff
@@ -170,17 +182,13 @@ func getNewConfigMap(sourceConfigMap *corev1.ConfigMap, namespace, sourceContext
 	destinationConfigMap := sourceConfigMap.DeepCopy()
 
 	destinationConfigMap.UID = ""
+	destinationConfigMap.Labels = map[string]string{}
```
Comment (mvleandro) [inline kube_syncer/abstractions/kubernetes_entity.go:185]: Only labels and annotations created by Keess should exist.

File: kube_syncer/abstractions/kubernetes_entity.go [hunk 5]
```diff
@@ -170,17 +182,13 @@ func getNewConfigMap(sourceConfigMap *corev1.ConfigMap, namespace, sourceContext
...
 	destinationConfigMap.Labels[ManagedLabelSelector] = "true"
+	destinationConfigMap.Annotations = map[string]string{}
```
Comment (mvleandro) [inline kube_syncer/abstractions/kubernetes_entity.go:187]: Only labels and annotations created by Keess should exist.

**Review by iblackman — APPROVED** [review]

File: kube_syncer/abstractions/kubernetes_entity.go [hunk 6]
```diff
@@ -190,17 +198,13 @@ func getNewSecret(sourceSecret *corev1.Secret, namespace, sourceContext string)
 	destinationSecret := sourceSecret.DeepCopy()
 
 	destinationSecret.UID = ""
+	destinationSecret.Labels = map[string]string{}
 	destinationSecret.Labels[ManagedLabelSelector] = "true"
+	destinationSecret.Annotations = map[string]string{}
```
Comment (mvleandro) [inline kube_syncer/abstractions/kubernetes_entity.go:203]: Only labels and annotations created by Keess should exist.

**Pull request merged**